### PR TITLE
Bug Fix - Re enable rageshake

### PR DESCRIPTION
**Pull request by Tyuoli**
https://github.com/vector-im/riot-android/issues/1971

### CODE REVIEW

File: matrix-sdk/src/main/java/org/matrix/androidsdk/sync/EventsThread.java [hunk 1]
```diff
@@ -156,6 +168,8 @@ public void setServerLongPollTimeout(int ms) {
 
     }
 
+
```
Comment (giomfo) [inline matrix-sdk/src/main/java/org/matrix/androidsdk/sync/EventsThread.java]: Do not add empty lines

File: matrix-sdk/src/main/java/org/matrix/androidsdk/MXSession.java [hunk 2]
```diff
@@ -159,6 +159,12 @@
     // tell if the data save mode is enabled
     private boolean mUseDataSaveMode;
 
+    // tell if the rageshake mode is enabled
+    public boolean mUseRageShakeMode;
+
+    // tell if the piwik mode is enabled
+    public boolean mUsePiwikMode;
```
Comment (giomfo) [inline matrix-sdk/src/main/java/org/matrix/androidsdk/MXSession.java]: This boolean is not used, please remove it

File: matrix-sdk/src/main/java/org/matrix/androidsdk/MXSession.java [hunk 3]
```diff
@@ -880,6 +886,7 @@ public void startEventStream(final EventsThreadListener anEventsListener, final
             }
 
             mEventsThread.setUseDataSaveMode(mUseDataSaveMode);
+            mEventsThread.setUseRageShakeMode(mUseRageShakeMode);
```
Comment (giomfo) [inline matrix-sdk/src/main/java/org/matrix/androidsdk/MXSession.java]: Why do you provide the rageshake mode to the events thread?

**Comment by Tyuoli** [comment]
Your remarks are relevant. So I changed my code so that the rageshake feature only applies to the app and not the SDK.

SDK should not be modified for the rageshake.

**Comment by bmarty** [comment]
Branch removed, PR has been rejected

**Pull request closed without merging**